components/self_hosted_purchase_modal/self_hosted_card.tsx

```diff
@@ -78,42 +73,14 @@ export default function SelfHostedCard(props: Props) {
             <Card
                 {...dummyCardProps}
                 intl={intl}
-                topColor='#4A69AC'
                 plan={props.desiredPlanName}
-                price={`${props.desiredProduct?.price_per_seat?.toString()}`}
                 seeHowBillingWorks={seeHowBillingWorks}
-                rate='/user/month'
-                planBriefing={<></>}
-                preButtonContent={(
-                    <SeatsCalculator
-                        price={props.desiredProduct?.price_per_seat}
-                        seats={props.seats}
-                        existingUsers={props.currentUsers}
-                        onChange={props.updateSeats}
-                    />
-                )}
-                afterButtonContent={<Consequences/>}
```

Comment:
I think we still want the wording from `Consequences`, so I think we still want afterButtonContent. It is similar, but also has details relevant to self-hosted about applying the license automatically.